src/TypeaheadMenu.react.js

```diff
@@ -19,7 +19,7 @@ class TypeaheadMenu extends React.Component {
     } = this.props;
 
     return (
-      <Menu {...menuProps}>
+      <Menu {...menuProps} optionCount={options.length}>
```

Comment:
`optionCount` doesn't appear to be used in `Menu`. Also, you should be able to get that value in `Menu` using `React.Children.count(this.props.children)`